tests/macros/test-CurrentGecko.js

```diff
@@ -0,0 +1,57 @@
+/* jshint node: true, mocha: true, esversion: 6 */
+
+var utils = require('./utils'),
```

Comment:
Out of interest, why `var` and not `const` here? We've used `const` in similar cases, e.g.: https://github.com/mdn/kumascript/blob/master/tests/macros/test-toolssidebar.js#L3.

Reply:
No good reason at all! 😄 I'll change it to use `const`, thanks @wbamberg!